docs/05-technical-reference/06-custom-resources/custrsrc-kyma-installation.md

```diff
@@ -34,8 +34,6 @@ spec:
     - name: "istio"
       namespace: "istio-system"
     - name: "provision-bundles"
-    - name: "dex"
-      namespace: "kyma-system"
     - name: "core"
```

Comment:
core can also be removed from here

Reply:
shouldn't this whole docu be removed? or at least replaced with describing components.yaml?